test/integration/host-mode-port-filter.bats

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	stop_manager
+	stop_docker
+}
+
+@test "docker should filter port in host mode correctly" {
+	start_docker 3
+	start_manager
+
+	#
+	# Use busybox to save image pulling time for integration test.
+	# Running the first 3 containers, it should be fine.
+	#
+	run docker_swarm run -d --expose=80 --net=host busybox sleep 30
```

Comment:
For containers that should run forever, perhaps `sh` would be a better command than `sleep`